.github/workflows/release-prod.yml

```diff
@@ -0,0 +1,39 @@
+name: Release (prod)
+
+on:
+  push:
+    branches: [main]
+    paths: ['modal/**']
```

Comment:
Any other paths we need to watch?

Reply:
We should add the poetry lock file as well